linux_os/guide/services/rng/service_rngd_enabled/rule.yml

```diff
@@ -0,0 +1,25 @@
+documentation_complete: true
+
+prodtype: rhel8
```

Comment:
You should add `ol8` and `fedora` here because you add the rule in Fedora and OL8 profile.

Reply:
Agreed with @jan-cerny but looks good so far. Nice job @adelton!

Reply:
Added, rebased on master -> 560ef9499bd31c684fb71796f56be479cb76d868.